.eslintrc.js

```diff
@@ -59,8 +59,12 @@ const restrictedImportPaths = [
     },
     {
         name: 'expensify-common',
-        importNames: ['Device'],
-        message: "Do not import Device directly, it's known to make VSCode's IntelliSense crash. Please import the desired module from `expensify-common/dist/Device` instead.",
+        importNames: ['Device', 'ExpensiMark'],
+        message: [
+            '',
+            "For 'Device', do not import it directly, it's known to make VSCode's IntelliSense crash. Please import the desired module from `expensify-common/dist/Device` instead.",
+            "For 'ExpensiMark', please use '@libs/Parser' instead.",
+        ].join('\n'),
```

Comment:
Wouldn't it be better, from the other dev perspective, to split this rule into 2 separate ones and get error only about the thing that was imported? WDYT? 
Device -> Device error message, ExpensiMark -> ExpensiMark error message